grub/grub.qubes-kernel-vm-support

```diff
@@ -6,3 +6,5 @@ if [ -r /usr/share/qubes/marker-vm ] &&
     GRUB_CMDLINE_LINUX="$GRUB_CMDLINE_LINUX xen_scrub_pages=0"
 fi
 GRUB_ENABLE_BLSCFG=false
+# Add qubes version to boot menu
+GRUB_DISTRIBUTOR="$(cat /etc/system-release)"
```

Comment:
Default grub config tries to keep it short, IMO even with version added, it still should be short - without `release`, and not including version twice (as it is in system-release file)

Reply:
I thought the same. Which one is more interesting? `Qubes release 4.2.2` or `Qubes (R4.2)`?
From technical point of view, the former is better. But the latter might be more interesting.
or maybe maybe make it like `Qubes (R4.2.2)`? It is necessary to decide if the minor version is useful or not.

Reply:
I wouldn't add minor version in the boot entry, I don't think boot entry title should change after installing ordinary updates. So make it `Qubes (R4.2)` or `Qubes R4.2`.

Reply:
Ok. I made it like this:
```
GRUB_DISTRIBUTOR="sed 's,\(.*\) release \(.*\) (R\(.*\)),\1 (R\2),g'" 
```

The 1st regex group catpures `Qubes`, the 2nd group captures `4.2.2` and the 3rd group captures `4.2`

The output is `Qubes (R4.2.2)`

It includes the minor version which is handy for debugging, troubleshooting and support matters. This is how I thought should be the best case. Please let me know if this fine or should be changed.